### DISCUSSION TITLE
peak2gene

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#1171] [Questions / Documentation] [answered]
Hi @rcorces 

I was wondering if the peak2gene correlation takes into account the TSS accessibility?

For example, if the inferred expression of gene X is correlated to the accessibility of region Y, does the p2g analysis take into account whether the TSS of gene X is accessible or not?

**Comment by rcorces** [comment, marked as answer]
No i do not believe that ArchR pays attention to TSS accessibility in peak2gene correlation
